### PR TITLE
merge me bitch

### PR DESCRIPTION
**Pull request by euanlivingstone**

### CODE REVIEW

File: app/views/visitors/index.html.slim [hunk 1]
```diff
@@ -27,17 +27,36 @@ section#services.services.bg-primary
...
+          .col-md-3.col-sm-6
+            .service-item
+              span.fa-stack.fa-4x
+                i.fa.fa-circle.fa-stack-2x
+                i.fa.fa-envelope-o.fa-stack-1x.text-primary
+              h4
+                strong Contact Us
+              = link_to courses_path, class: "btn btn-default"
+                | View our Contact Us
+- if user_signed_in?
```
Comment (JamesSharkey) [inline app/views/visitors/index.html.slim]: should be if ! or unless.

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.2.1
+2.2.3
```
Comment (JamesSharkey) [inline .ruby-version]: change back to 2.2.1

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -18,8 +18,8 @@ html
         button.close aria-label="Close" data-dismiss="alert" type="button"
           span aria-hidden="true" &times;
         = alert
-    = nav_bar fixed: :top, brand: "NMT" do 
-      = menu_group do 
+    = nav_bar fixed: :top, brand: image_tag("NMT-logo3.png", size: "80") do
```
Comment (JamesSharkey) [inline app/views/layouts/application.html.slim]: image_tag("NMT-logo3.png", size: "80", class: "pts")

File: app/views/visitors/index.html.slim (hunk 1 above)
Comment (euanlivingstone) [inline app/views/visitors/index.html.slim]: I copied and pasted your code from another page!

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -18,8 +18,8 @@ html
         button.close aria-label="Close" data-dismiss="alert" type="button"
           span aria-hidden="true" &times;
         = alert
-    = nav_bar fixed: :top, brand: "NMT" do 
-      = menu_group do 
+    = nav_bar fixed: :top, brand: image_tag("NMT-logo3.png", size: "80", class: "pts") do
```
Comment (JamesSharkey) [inline app/views/layouts/application.html.slim]: i dont want the img to hang off the nav like that lower the size so it looks centered on the nav

Comment (euanlivingstone) [inline app/views/layouts/application.html.slim]: it does not look centred but is not over hanging now

**Pull request merged**